### ISSUE TITLE
Canary v128: e.canCreateGenericSession is not a function

### ISSUE THREAD

**Issue by TomTom101** [nicoalbanese/ai-sdk-chrome-ai#7]
`await window.ai.canCreateTextSession()` in the console returns `readily`

**Comment by koolkao** [comment]
Same issue here. I'm running Version 128.0.6595.0 (Official Build) canary (arm64) on MacOS 

Error: 
TypeError: e.canCreateGenericSession is not a function
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36

**Comment by R2turnTrue** [comment]
Same issue yet!

128.0.6609.0 (Official Build) canary (64bit) (cohort: Clang-64) on Windows
1c4b506ddabf822681f10d673b7018a8c9e519ab-refs/branch-heads/6609@{#1}

Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36

```
TypeError: e.canCreateGenericSession is not a function
    at c (page-27e34efddc4a8a89.js:1:760)
    at page-27e34efddc4a8a89.js:1:15662
    at page-27e34efddc4a8a89.js:1:15748
    at aW (67cfe1a8-7e7bb0bc1c7f5493.js:1:73241)
    at oe (67cfe1a8-7e7bb0bc1c7f5493.js:1:84682)
    at ol (67cfe1a8-7e7bb0bc1c7f5493.js:1:85320)
    at or (67cfe1a8-7e7bb0bc1c7f5493.js:1:85204)
    at ol (67cfe1a8-7e7bb0bc1c7f5493.js:1:85300)
    at or (67cfe1a8-7e7bb0bc1c7f5493.js:1:85204)
    at ol (67cfe1a8-7e7bb0bc1c7f5493.js:1:86265)
```

**Comment by craigz** [comment]
running the dev build on an m1 mac, chrome version: `Version 129.0.6614.3 (Official Build) dev (arm64)` and am seeing the same thing. in the js console entering `window.ai` shows the prototype definition as follows:

```
AI {}
[[Prototype]]: AI
  canCreateTextSession: ƒ canCreateTextSession()
  createTextSession: ƒ createTextSession()
  defaultTextSessionOptions: ƒ defaultTextSessionOptions()
  constructor: ƒ AI()
  Symbol(Symbol.toStringTag): "AI"
  [[Prototype]]: Object
```

that said, from the console, i can create a new session with `ai.createTextSession();` and then go ahead and 'converse' with the model via `await session.prompt("some question for the local model?");` and will receive a reply, which for the several simple questions asked was correct. 

perhaps the demo can function with a change to `createTextSession()`?

**Comment by BogdanManescu** [comment]
The issue is in chromeai, there is an error in the declaration or calling of ai.defaultTextSessionOptions

**Comment by nicoalbanese** [comment]
resolved with #8!
